.github/workflows/py-build.yml

```diff
@@ -5,6 +5,7 @@ on:
     paths:
       - "api/python/**"
       - "tools/cell_census_builder/**"
+      - ".github/workflows/**"
```

Comment:
This is useful in case one wants to modify the workflow and test it immediately - we can either remove and keep it.

Reply:
OK w/ me if we keep it.

Reply:
maybe comment it out with a explanation of why it's useful.